javascript/javascript3/redo/week2/promise.js

```diff
@@ -0,0 +1,23 @@
+function resolveAfterTime(resolveAfter) {
+    return new Promise((resolve) => setTimeout(() => {
```

Comment:
Nitpick(code style): You could reduce the Promise body to `setTimeout(resolve)` if you wanted to.